packages/card/index.js

```diff
@@ -0,0 +1,116 @@
+import { FabricWebComponent } from '../utils';
+import { card as c } from '@fabric-ds/component-classes';
+import { classNames } from '@chbphone55/classnames';
+
+class FabricCard extends FabricWebComponent {
+  static get observedAttributes() {
+    return ['onlick', 'selected'];
+  }
+
+  attributeChangedCallback(name, _, newValue) {
+    switch (name) {
+      case 'onclick':
```

Comment:
Do we need to handle this ourselves? Wouldn’t we get onclick for free?

Reply:
This is true, I was under the impression that it returned a string and not actually a callable function, but seems to be working without this.